### PR TITLE
Fix interface default issues

### PR DESCRIPTION
**Pull request by saichint**
When the interface defaults were defined, there are few issues creeped in.
1. couple of typos
2. test issues where the interface needs to be put in switchport_mode for the corresponding properties to test.

### CODE REVIEW

File: tests/test_interface.rb [hunk 1]
```diff
@@ -1328,8 +1330,6 @@ def interface_test_data
         default_shutdown:    true,
         switchport:          :disabled,
         default_switchport:  :disabled,
-        access_vlan:         DEFAULT_IF_ACCESS_VLAN,
```
Comment (mikewiebe) [inline tests/test_interface.rb:1331]: Why are you removing these?

**Comment by mikewiebe** [comment]
@saichint One comment, otherwise 👍

**Comment by mikewiebe** [comment]
@saichint Please add a note indicating which platforms/image versions you tested.

File: tests/test_interface.rb (hunk 1 above)
Comment (saichint) [reply tests/test_interface.rb:1331]: For loopback and vlan interfaces, access_vlan is not an option. For ethernet and port-channel it is available if the switchport mode is turned on.

**Comment by saichint** [comment]
Tested on n9k running 9.2.1 image and on n7k running 8.3.1 image

**Pull request merged**